src/main/java/core/basesyntax/model/machine/Car.java

```diff
@@ -1,5 +1,8 @@
 package core.basesyntax.model.machine;
 
+import javax.persistence.Entity;
+
+@Entity
 public class Car extends Machine {
     private int horsePower;
```

Comment:
```suggestion
    private int horsePower;
```
better name this column in db like horse_power)